tests/cluster/tests/helpers/onlydots.tcl

```diff
@@ -1,5 +1,5 @@
 # Read the standard input and only shows dots in the output, filtering out
-# all the other characters. Designed to avoid bufferization so that when
+# all the other characters. Designed to avoid buffering so that when
```

Comment:
I'm hoping that this is ok, but I wouldn't be shocked if I'm wrong.

Reply:
I think 'bufferization' is correct - it's about being turned into a buffer, not the buffer itself.